python/graphscope/framework/app.py

```diff
@@ -312,7 +312,7 @@ class name.
         self._session.dag.add_op(self._op)
 
     def __repr__(self):
-        s = f"graphscope.App <type: {self._app_assets.type}, algorithm: {self._app_assets.algo}"
+        s = f"graphscope.App <type: {self._app_assets.type}, algorithm: {self._app_assets.algo} "
```

Comment:
Why we need a trailing space ` ` here?

Reply:
There is a string `add` operation in `__repr__` function.
```
def __repr__(self):
    s = f"graphscope.App <type: {self._app_assets.type}, algorithm: {self._app_assets.algo} "
    s += f"bounded_graph: {str(self._graph)}>"
    return s
```